ui/src/dashboards/actions/creators.ts

```diff
@@ -0,0 +1,84 @@
+// Types
+import {Dashboard, Label, RemoteDataState, DashboardEntities} from 'src/types'
+import {NormalizedSchema} from 'normalizr'
+
+export const SET_DASHBOARDS = 'SET_DASHBOARDS'
+export const SET_DASHBOARD = 'SET_DASHBOARD'
+export const REMOVE_DASHBOARD = 'REMOVE_DASHBOARD'
+export const DELETE_DASHBOARD_FAILED = 'DELETE_DASHBOARD_FAILED'
+export const EDIT_DASHBOARD = 'EDIT_DASHBOARD'
+export const REMOVE_CELL = 'REMOVE_CELL'
+export const ADD_DASHBOARD_LABEL = 'ADD_DASHBOARD_LABEL'
+export const REMOVE_DASHBOARD_LABEL = 'REMOVE_DASHBOARD_LABEL'
```

Comment:
what do you think about alphabetizing these? not sure how to do it in the vastly superior vscode, but in sublime, you highlight a block of code and hit `f5`.

Reply:
hahah.  i think its `f9` in vscode.